tests/acceptance/01_vars/02_functions/string_split_2k_variable.cf

```diff
@@ -0,0 +1,13 @@
+
+    "test_list" -> { "CFE-3047" } # Promise to ticket that we don't crash
+      slist => string_split("$(test_str)", "\n", "10000" );
+
+  reports:
+    any::
+      "$(this.promise_filename) Pass";
```

Comment:
I think it this should at least have a comment showing what the expected result is.